apps/pie-storybook/stories/testing/pie-chip.test.stories.ts

```diff
@@ -86,13 +86,14 @@ const chipStoryMeta: ChipStoryMeta = {
 
 export default chipStoryMeta;
 
+/* eslint-disable no-console */
 const clickAction = () => {
```

Comment:
is there a way to disable these warnings?

Reply:
We probably should be emitting events and using Playwrights functions for event listeners. Do we have a ticket for this, either a spike or a task?

Reply:
Created as part of DSW-2680